server/public/model/feature_flags.go

```diff
@@ -78,6 +78,10 @@ type FeatureFlags struct {
 	// FEATURE_FLAG_REMOVAL: ChannelAdminManageABACRules - Remove this field when feature is GA
 	// Enable channel admins to manage ABAC rules for their channels
 	ChannelAdminManageABACRules bool
+
+	// FEATURE_FLAG_REMOVAL: AutoTranslate - Remove this default when MVP is to be released
```

Comment:
Ultra-nit: this is not the default, but the declaration itself.
```suggestion
	// FEATURE_FLAG_REMOVAL: AutoTranslate - Remove this when MVP is to be released
```